# Parameters for adaptive_grasping package
# Remember that there can be other important constants that can only be changed
# from the src code.

robot_commander:
  # Enable reference lowpass filtering
  enable_filter: false
  # Velocity upper limit (in absolute value)
  vel_limit: 0.3
  

